transport-classes-io_uring/src/main/java/io/netty/channel/uring/IoUringIoHandler.java

```diff
@@ -396,6 +406,13 @@ public boolean isCompatible(Class<? extends IoHandle> handleType) {
         return IoUringIoHandle.class.isAssignableFrom(handleType);
     }
 
+    public void adjustIOWQWorkder(int maxBoundedWorker, int maxUnboundedWorker) {
```

Comment:
Do we really need to be able to adjust this on the fly ?

This also might cause problems as people might call this even when the ring is already destroyed

Reply:
I will remove it